### DISCUSSION TITLE
Redirects for renamed slugs

### DISCUSSION THREAD

**Discussion by Gaming32** [modrinth/.github#101] [Feature requests]
If you change a slug, there should be an option to setup a redirect from the old slug to the new slug. The redirect would apply to both Knossos and the API. It could also maybe show a notice that you were redirected. The original slug would be "loosely claimed", so that if any new project uses that old slug, the new project gets that slug without issue, replacing the redirect.

**Comment by IMB11** [comment]
Maybe lock the slug for a week, then send a notification to the author saying the slug will still redirect but can be taken by anyone now.
And when the slug is re-taken and the redirect removed, send a notification saying the redirect is no longer in action.

**Reply by triphora** [reply]
In addition to this, there would probably also need to be a time limit, like Minecraft accounts, where a project could only have its name changed every week in order to prevent abuse.
